tests/frontends/test_pytorch.py

```diff
@@ -983,6 +988,127 @@ def step(model, inp):
     #assert torch.allclose(my_out_db_grad, bias.grad.data)
 #'''
 
+'''
+def test_nn_conv2d_fwd():
+    backend = 'pytorch'
+    backend_options = get_backend_options(args, backend)
+
+    torch.manual_seed(123)
+
+    input = torch.randn(2,6,4,5, dtype=getattr(torch, args.dtype))
+
+    class Conv2dMod(nn.Module):
+        def __init__(self):
+            super(Conv2dMod, self).__init__()
+            self.conv1 = nn.Conv2d(6, 3, (4, 5), (2,3), (3,2), (3,4), 3)
+
+        def forward(self, inp):
+            return self.conv1(inp)
+
+    def cost(model, inp):
+        value = model(inp)
+        return torch.sum(value)
+
+    model = Conv2dMod()
+
+    #"""
+    @myia(backend=backend, backend_options=backend_options)
+    def step(model, inp):
+    #def step(inp):
+        return model(inp)
+    #out = step(input)
+    out = step(model, input)
+    #loss, model = step(model, input)
+    #"""
+
+    #pt_cost = cost(model, input)
+    pt_cost = model(input)
+    """
+    if input.grad is not None:
+        input.grad.data.zero_()
+    if weight.grad is not None:
+        weight.grad.data.zero_()
+    if bias.grad is not None:
+        bias.grad.data.zero_()
+
+    pt_cost.backward()
+    #"""
+
+    #assert torch.allclose(my_out_dinp_grad, input.grad.data)
+    #assert torch.allclose(my_out_dw_grad, weight.grad.data)
+    #assert torch.allclose(my_out_db_grad, bias.grad.data)
+#'''
```

Comment:
Remove comment.

Reply:
This is going to be used in next PR when Heterogeneous broaden.

Reply:
I would prefer it if you stashed it or committed it on a different branch, so that the master version remains clean.